### PR TITLE
Added new sectionto brand(MeshMap) #3629

### LINKED ISSUE #3629: [Brand] Add MeshMap brand kit
#### Current Behavior
https://layer5.io/brand has all of the project brand kits, but is missing MeshMap.

#### Desired Behavior
Create a new MeshMap section in the brand page.

#### Implementation
1. Create a new section for MeshMap
2. Add the following .zip as a download - 
[meshmap-brand-kit.zip](https://github.com/layer5io/layer5/files/10429792/meshmap-brand-kit.zip)

3. Add a few of the logos as samples.

#### Acceptance Tests
1. Users can see the new section.
2. Users can click on the intra-page nav menu to be directed to the in-page anchor for this new section.
3. Users are pushed a .zip file containing the brand-kit on click of the download button.

---
<h3> Contributor Resources and <a href="https://layer5.io/community/handbook">Handbook</a></h3>

The layer5.io website uses Gatsby, React, and GitHub Pages. Site content is found under the [`master` branch](https://github.com/layer5io/layer5/tree/master).
- 📚 See [contributing instructions](https://github.com/layer5io/layer5/blob/master/CONTRIBUTING.md)
- 🎨 Wireframes and designs for Layer5 site in [Figma](https://www.figma.com/file/5ZwEkSJwUPitURD59YHMEN/Layer5-Designs). 
- 🙋🏾🙋🏼 Questions: [Discussion Forum](https://discuss.layer5.io) and [Community Slack](http://slack.layer5.io)

### PR DESCRIPTION
**Pull request by YarinDay**
**Description**

This PR fixes #3629 

**Notes for Reviewers**

# Before:
image

# After:
image

**[Signed commits](https://github.com/layer5io/layer5/blob/master/CONTRIBUTING.md#signing-off-on-commits-developer-certificate-of-origin)**
- [x] Yes, I signed my commits.

### CODE REVIEW

**Comment by YarinDay** [comment]
@leecalcote Hey bro, I added the section you wanted to add, would like you to review it :)

**Comment by Nikhil-Ladha** [comment]
Please update the PR title to a better phrase.

**Comment by l5io** [comment]
🚀 Preview for commit 261e45815c3c493b9afc8b8c9de4a2f7138fd9a8 at: https://63cbdae0431b306e69db4479--layer5.netlify.app

File: src/sections/Company/Brand/Brand-components/meshmap.js [hunk 1]
```diff
@@ -0,0 +1,62 @@
+import React from "react";
+import { SRLWrapper } from "simple-react-lightbox";
+import { Row, Col } from "../../../../reusecore/Layout";
+import Button from "../../../../reusecore/Button";
+import { FiDownloadCloud } from "@react-icons/all-files/fi/FiDownloadCloud";
+import MeshMapIconStacked from "../../../../assets/images/meshmap/meshmap-color-stacked.png";
+import MeshMapIconHorizontal from "../../../../assets/images/meshmap/meshmap-color-horizontal.svg";
+import MeshMapText from "../../../../assets/images/meshmap/meshmap-color-text.png";
```
Comment (leecalcote) [inline src/sections/Company/Brand/Brand-components/meshmap.js:8]: Thanks for this, @YarinDay. Will you consider only displaying the SVG versions?

**Comment by leecalcote** [comment]
Nice work, @YarinDay. You even managed to include the .zip in the two appropriate locations.

**Review by leecalcote — CHANGES_REQUESTED** [review]

File: src/sections/Company/Brand/Brand-components/meshmap.js (hunk 1 above)
Comment (YarinDay) [reply src/sections/Company/Brand/Brand-components/meshmap.js:8]: @leecalcote Yeah of course :), I'm on it!

Comment (YarinDay) [reply src/sections/Company/Brand/Brand-components/meshmap.js:8]: # Before
image


# After
image

**Comment by l5io** [comment]
🚀 Preview for commit 15cab04d70e230fe4ecf4ce7a0c05c5affab4219 at: https://63cc1e8afe08a62b8a616471--layer5.netlify.app

**Comment by Nikhil-Ladha** [comment]
Hi @YarinDay , I see the issue is assigned to someone else and you submitted the PR without any prior ask. Please refrain from doing this in the future, it helps avoid duplicate work and creates less confusion. I won't be considering this PR since the person who is the actual assignee of the issue has already made a PR.
Hopefully, you understand the situation here :)

**Comment by leecalcote** [comment]
merge conflicts here...

@YarinDay would you like to address those?

**Comment by YarinDay** [comment]
> merge conflicts here...
> 
> @YarinDay would you like to address those?

Yea, I can do it

**Comment by Nikhil-Ladha** [comment]
Closing since this has already been fixed

**Pull request closed without merging**